client/equo-completion.zsh

```diff
@@ -12,7 +12,11 @@ _equo_get_mirrors()
 
 _equo_get_cmds()
 {
-  cmds=( ${(f)"$(equo $1 --help | sed 's/--multifetch/--multifetch  can be/' | sed -r -e '/^(positional|action:)/ {N; d;}' | grep -P '^  (?!-h|<package>|{|   )' | sed -r 's/^ {2,4}(-*[a-zA-z0-9-]+)(, -\w|)  +(\w.*)/\1:\3/')"} )
+  cmds=( ${(f)"$(equo $1 --help |
```

Comment:
Sorry, I commented on the old line. This is the new line. It's not too long surely?